draft-bbs-signatures.md

```diff
@@ -346,18 +351,23 @@ Procedure:
 
 Sign computes a signature from SK, PK, over a vector of messages. This method
 describes deterministic signing. For threshold signing, XOF can be replaced
-with a PRF due to the insecurity of deterministic threshold signing.
+with a PRF due to the insecurity of deterministic threshold signing. 
 
 ```
 signature = Sign(SK, PK, (msg_1,..., msg_L), (H_1,..., H_L))
 
 Inputs:
 
-- msg_1,...,msg_L, octet strings. Messages to be signed.
+- msg_1,..., msg_L, octet strings. Messages to be signed.
 - H_1,..., H_L, points of G1. Generators used to sign the messages.
 - SK, a secret key output from KeyGen
 - PK, a public key output from SkToPk
 
+Parameters:
```

Comment:
Do h_s and h_d now have to listed as parameters here?

Reply:
IMO not technically since they are defined in the terminology and are not optional. We could add them for clarity in the parameters also though.

Reply:
Ok so we are saying that parameters differ from inputs in that they are optional? IMO I think it is clearer to have them defined here also.

Reply:
Done.